lib/chef/chef_fs.rb

```diff
@@ -53,7 +53,7 @@
 class Chef
   module ChefFS
     def self.windows?
```

Comment:
can we mark this as deprecated and kill it off in the future as well?

Reply:
that ChefFS API isn't a public method, it should just get cleaned up.  out of scope though.